src/commands/commerce/store/quickstart/setup.ts

```diff
@@ -488,6 +495,144 @@ export class StoreQuickstartSetup extends SfdxCommand {
         );
     }
 
+    private async initShipping(): Promise<void> {
+        // retrieve webStoreId
+        const webStoreId = forceDataSoql(
+            `SELECT Id FROM WebStore WHERE Name='${this.varargs['communityNetworkName'] as string}' LIMIT 1`,
+            this.org.getUsername(),
+            this.flags,
+            this.logger
+        ).result.records[0].Id;
+
+        // Retrieve the application context from WebStore
+        const applicationContext = await this.getApplicationContextFromWebStoreId(webStoreId);
+        this.ux.log(`Retrieved Application Context for WebStore : '${JSON.stringify(applicationContext)}'`);
+
+        // Retrieve the Currency Settings for the Org
+        const currencySettings = await this.getCurrencySettings();
+        this.ux.log(`Retrieved CurrencySettings for org : '${JSON.stringify(currencySettings)}'`);
+
+        // Creating Shipping Configuration Set
+        forceDataRecordCreate(
+            'ShippingConfigurationSet',
+            `TargetRecordId=${webStoreId}`,
+            this.org.getUsername(),
+            this.flags,
+            this.logger
+        );
+
+        const configurationSetId = forceDataSoql(
+            `SELECT Id FROM ShippingConfigurationSet WHERE TargetRecordId='${webStoreId}' LIMIT 1`,
+            this.org.getUsername(),
+            this.flags,
+            this.logger
+        ).result.records[0].Id;
+
+        this.ux.log(`Created shippingConfigurationSet with id : '${configurationSetId}'`);
+
+        // Creating ShippingRateGroup
+        forceDataRecordCreate(
+            'ShippingRateGroup',
+            `ShippingProfileId=${configurationSetId}`,
+            this.org.getUsername(),
+            this.flags,
+            this.logger
+        );
+
+        const shippingRateGroupId = forceDataSoql(
+            `SELECT Id FROM ShippingRateGroup WHERE ShippingProfileId='${configurationSetId}'`,
+            this.org.getUsername(),
+            this.flags,
+            this.logger
+        ).result.records[0].Id;
+
+        this.ux.log(`Created ShippingRateGroup with id : '${shippingRateGroupId}'`);
+
+        // Creating ShippingRateArea
+        forceDataRecordCreate(
+            'ShippingRateArea',
+            `ShippingRateGroupId=${shippingRateGroupId} Countries='${applicationContext['country']}'`,
+            this.org.getUsername(),
+            this.flags,
+            this.logger
+        );
+
+        const shippingZoneId = forceDataSoql(
+            `SELECT Id from ShippingRateArea WHERE ShippingRateGroupId='${shippingRateGroupId}'`,
+            this.org.getUsername(),
+            this.flags,
+            this.logger
+        ).result.records[0].Id;
+
+        this.ux.log(`Created ShippingRateArea with id : '${shippingZoneId}'`);
+
+        // Creating StandardShippingArea
+        const isMultiCurrencyEnabled = currencySettings['records'][0]['IsMultiCurrencyEnabled'];
+
+        if (isMultiCurrencyEnabled) {
```

Comment:
Can we add tests for these two code paths?

Reply:
Just to confirm [this](https://github.com/forcedotcom/commerce-on-lightning/blob/develop/test/commands/commerce/store/quickstart/setup.test.ts) is where test should be added right ?

Reply:
Yes 👍

Reply:
Done.